dask_ml/model_selection/_search.py

```diff
@@ -42,28 +43,31 @@
     cv_extract_params,
     cv_n_samples,
     cv_split,
-    decompress_params,
     feature_union,
     feature_union_concat,
     fit,
     fit_and_score,
     fit_best,
     fit_transform,
-    get_best_params,
     pipeline,
     score,
 )
 from .utils import DeprecationDict, is_dask_collection, to_indexable, to_keys, unzip
 
+logger = logging.getLogger(__name__)
+
 try:
     from cytoolz import get, pluck
 except ImportError:  # pragma: no cover
     from toolz import get, pluck
 
+try:
+    from dask.distributed import as_completed
```

Comment:
Is this try/except needed? `distributed` is a dependency now (#466).

Reply:
I'm fine with removing the optional check.  Previous guidance was to make it option as it wasn't a dependency at the time.